assets/helpers/stripeCheckout/stripeCheckout.js

```diff
@@ -52,12 +52,20 @@ export const setupStripeCheckout = (
   });
 });
 
-export const openDialogBox = (amount: number, email: string) => {
-  if (stripeHandler) {
+export const openDialogBox = (
+  amount: number,
+  email: string,
+  postDeployTestCallback: Function,
+  isPostDeploymentTestUser: boolean,
+) => {
+  if (stripeHandler && !isPostDeploymentTestUser) {
     stripeHandler.open({
       // Must be passed in pence.
       amount: amount * 100,
       email,
     });
+  } else if (stripeHandler && isPostDeploymentTestUser) {
```

Comment:
Since in both case it is being checking for `stripeHandler`, I would extract that condition in a separate if at the beginning. 

```javascript
if(stripeHandler) {
    if(isPostDeploymentTestUser) {
       postDeployTestCallback('tok_visa');
    } else {
        stripeHandler.open({
            // Must be passed in pence.
            amount: amount * 100,
            email,
    }
}
```